### DISCUSSION TITLE
'use client' causes page to crash, in dev but not in build

### DISCUSSION THREAD

**Discussion by jmuffat** [vercel/next.js#52601] [Help] [answered]
### Summary

I am starting a new project and am going for the app router this time.
I could go reasonably far, until I added my first 'useState' ...

Page quickly flashes and then crashes during hydration with `TypeError: __webpack_require__.n is not a function`

This problem only happens with `npm run dev`, if I `npm run build` then `npm start`, I get no issue.

At this stage, I'm using the counter code at https://nextjs.org/docs/getting-started/react-essentials and adding that component to my page that otherwise works fine. 

I am running out of ideas, any hint ?

### Additional information

```js
Operating System:
      Platform: darwin
      Arch: arm64
      Version: Darwin Kernel Version 22.5.0: Thu Jun  8 22:21:34 PDT 2023; root:xnu-8796.121.3~7/RELEASE_ARM64_T8112
    Binaries:
      Node: 18.16.1
      npm: 9.5.1
      Yarn: N/A
      pnpm: N/A
    Relevant Packages:
      next: 13.4.9
      eslint-config-next: 13.4.9
      react: 18.2.0
      react-dom: 18.2.0
      typescript: 5.1.6
    Next.js Config:
      output: N/A
```

**Comment by jmuffat** [comment, marked as answer]
Ok, I should have tried this before posting : 

_The problem happens on Safari but doesn't on Chrome._

I'm fine working with Chrome, so this is a non issue.

**Comment by xkisu** [comment]
I'm experiencing this as well on Safari, is there a solution to it aside from developing on Chrome instead?

**Reply by jmuffat** [reply]
I'm really not sure what is happening. I thought it was the `(app-client)` bit that Safari didn't understand like in :
```
    "(app-client)/./node_modules/next/dist/build/polyfills/polyfill-module.js":
```

Now, today, I try and use Safari again to copy-paste the actual place where the problem occurs and... it doesn't occur any more ?!!
The only thing that changed that I can think of is that I applied the `OSX Ventura 13.4.1(c)` update.

I'll try and switch back to Safari, see if I still get any issue.
